app/src/main/java/com/cornellappdev/android/eatery/presenter/MainListPresenter.java

```diff
@@ -84,6 +96,40 @@ private boolean isUnderCategory(EateryBaseModel model) {
         return cafesToDisplay;
     }
 
+    public void sortNearestFirst(Context context) {
+        // It's fine to have this code in presenter, right?
```

Comment:
wait for lesley on this one, and then delete this before you merge

Reply:
yes, this is the right move. all logic based code should go in our presenters.